lib/sqlalchemy/sql/sqltypes.py

```diff
@@ -604,25 +604,6 @@ class Numeric(_LookupExpressionAdapter, TypeEngine):
         type or a subclass, otherwise numeric coercion between
         ``float``/``Decimal`` may or may not function as expected.
 
-    .. note::
-
-       The Python ``decimal.Decimal`` class is generally slow
-       performing; cPython 3.3 has now switched to use the `cdecimal
-       <http://pypi.python.org/pypi/cdecimal/>`_ library natively. For
-       older Python versions, the ``cdecimal`` library can be patched
-       into any application where it will replace the ``decimal``
-       library fully, however this needs to be applied globally and
-       before any other modules have been imported, as follows::
-
-           import sys
-           import cdecimal
-           sys.modules["decimal"] = cdecimal
-
-       Note that the ``cdecimal`` and ``decimal`` libraries are **not
-       compatible with each other**, so patching ``cdecimal`` at the
-       global level is the only way it can be used effectively with
-       various DBAPIs that hardcode to import the ``decimal`` library.
-
```

Comment:
we still support py2.7. was it updated there also?

Reply:
It wasn't. Even with 2.7 still supported I think it's worth dropping this now over keeping it in and forgetting to drop it later.